eventstore/eventstore.go

```diff
@@ -106,7 +106,15 @@ func migrate(store EventStore) error {
 	})
 }
 
-func getEventsToMigate(db *bolt.DB) ([]Event, []EventTimes, error) {
+// copyBytes gets used to copy the keys and value from boltdb to prevent
+// 'unexpected fault address' errors.
+func copyBytes(b []byte) []byte {
+	out := make([]byte, len(b))
+	copy(out, b)
+	return out
+}
+
+func getEventsToMigate(db *bolt.DB) ([]Event, *[]EventTimes, error) {
```

Comment:
Why is a pointer to a slice being returned here? That's almost certainly unnecessary.